### DISCUSSION TITLE
Am I missing something??

### DISCUSSION THREAD

**Discussion by joshv** [nuxt/nuxt#16211] [Questions]
I am trying to create the simplest example and it doesn't work..  I am literally trying to use the example from the documentation for useRequestEvent, https://nuxt.com/docs/api/composables/use-request-event.  My page code is below.  When the page loads in the browser I get the error, "Cannot read properties of undefined (reading 'node')".  I have written the event object out to the console and node is definitely not undefined.  What am I missing?

```
<script setup>
  
  const event = useRequestEvent();
  console.log(event);
  const url = event.node.req.url;

</script>

<template>
  <div id="app">
    URL {{ event.node.req.url }}
  </div>
</template>
```

**Comment by olivervorasai** [comment]
The doc says "In the browser, `useRequestEvent` will return `undefined`."

Perhaps the docs should be more clear but you can use that composable like this:

```javascript
const { ssrContext } = useNuxtApp();

const event = useRequestEvent();

if (ssrContext) {
    console.log(event);
}
```

The output will be in your terminal console, not the browser console.
